pgrx/src/callconv.rs

```diff
@@ -262,7 +262,7 @@ macro_rules! argue_from_datum {
 argue_from_datum! { 'fcx; i8, i16, i32, i64, f32, f64, bool, char, String, Vec<u8> }
 argue_from_datum! { 'fcx; Date, Interval, Time, TimeWithTimeZone, Timestamp, TimestampWithTimeZone }
 argue_from_datum! { 'fcx; AnyArray, AnyElement, AnyNumeric }
-argue_from_datum! { 'fcx; Inet, Internal, Json, JsonB, Uuid }
+argue_from_datum! { 'fcx; Inet, Internal, Json, JsonB, Uuid, crate::rel::PgRelation }
```

Comment:
Also, I would prefer this was actually imported, because I had to spend _considerable_ amounts of time refactoring messy imports recently, and I have developed a Strong Preference around actual imports.

specifically `module::Type` is okay by me but I would prefer to avoid from the crate root except in the top level imports.

Reply:
Done.